libcobblersignatures/enums.py

```diff
@@ -0,0 +1,88 @@
+"""
+This module contains all enums which are used in the library.
+"""
+
+from enum import Enum
+
+
+class OsArchitectures(Enum):
+    """
+    An enumeration which defines the in Cobbler available architectures.
+    """
+    i386 = 1
+    """
+    32-bit architecture which is also called ``IA-32`` or ``80x86`` by some people.
+    """
+    x86_64 = 2
```

Comment:
[nitpick] Personally I think this looks kind of weird, what is the advantage over just using comments?

Reply:
These docstring get published in rtd. If I am doing normal comments they don't get published on rtd.

> rtd = Read The Docs